aptos-move/aptos-debugger/src/aptos_debugger.rs

```diff
@@ -133,7 +134,7 @@ impl AptosDebugger {
 
                     // Deprecated.
                     TransactionPayload::ModuleBundle(..) => {
-                        unreachable!("Module bundle payload has already been checked")
+                        unreachable!("Module bundle payload has already been deprecated")
```

Comment:
This messages said checked because before this function is called, we already reject module bundles. So it is truly unreachable code because of the check

Reply:
I see I didn't realize there's a check before this. Will revert it back but maybe add a bit more context